src/components/comparePricing/comparePricing.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react'
+import stl from './comparePricing.module.scss'
+import PricingCard from './pricingCard'
+import {
+  Nigeria,
+  India,
+  Mexico,
+  CostaRica,
+  Colombia,
+  Dominican,
+} from 'assets/svg'
+
+const ComparePricing = () => {
+  const [data, setData] = useState('')
+
+  const countries = {
+    Nigeria: {
+      average: '62',
+      physician: '35',
+      psychologist: '3',
+      nutritionist: '24',
+    },
+    India: {
+      average: '82',
+      physician: '45',
+      psychologist: '13',
+      nutritionist: '24',
+    },
+    Mexico: {
+      average: '85',
+      physician: '45',
+      psychologist: '16',
+      nutritionist: '24',
+    },
+    CostaRica: {
+      average: '81',
+      physician: '45',
+      psychologist: '12',
+      nutritionist: '24',
+    },
+    Colombia: {
+      average: '80',
+      physician: '40',
+      psychologist: '16',
+      nutritionist: '24',
+    },
+    Dominican: {
+      average: '75',
+      physician: '45',
+      psychologist: '16',
+      nutritionist: '14',
+    },
+  }
+  const parentToChild = country => {
+    data = countries[country]
+    setData(data)
+  }
+
+  return (
+    <section className={stl.container}>
+      <div className={stl.pricing}>
+        <div className={stl.countries}>
+          <div className={stl.country} onClick={() => parentToChild('Nigeria')}>
```

Comment:
Maybe render the countries list using for in loop or a map() method by converting the countries object to an array of objects. As this is not scalable if a new country is added then we need to also add a new div with all the styles. With map() or loop and rendering dynamically we can change that to make it easier to parse and scale.